### PR TITLE
chore: gitpodify

### PR DESCRIPTION
**Pull request by ghostdevv**
I added an initial gitpod config which will install via npm and add the eslint extension - I'd love to know if there is anything else people usually run when they first jump in to work on a new feature (for example a playground environment) so I can also run that.

I also added an "Open in Gitpod" and "Contribute with Gitpod" button in two places - LMK if there is anywhere I missed

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

File: .gitpod.yml
Comment (Eomm) [inline .gitpod.yml:1]: could it be added to .npmignore ?

File: README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ downloads](https://img.shields.io/npm/dm/fastify.svg?style=flat)](https://www.np
 [![Security Responsible
 Disclosure](https://img.shields.io/badge/Security-Responsible%20Disclosure-yellow.svg)](https://github.com/fastify/fastify/blob/main/SECURITY.md)
 [![Discord](https://img.shields.io/discord/725613461949906985)](https://discord.gg/fastify)
+[![Contribute with Gitpod](https://img.shields.io/badge/Contribute%20with-Gitpod-908a85?logo=gitpod)](https://gitpod.io/#https://github.com/fastify/fastify)
```
Comment (Eomm) [inline README.md:31]: Gray tells me that something is not working TBH

```suggestion
[![Contribute with Gitpod](https://img.shields.io/badge/Contribute%20with-Gitpod-908a85?logo=gitpod&color=blue)](https://gitpod.io/#https://github.com/fastify/fastify)
```

File: .gitpod.yml
Comment (ghostdevv) [reply .gitpod.yml:1]: Done

**Comment by ghostdevv** [comment]
Updated

**Review by metcoder95 — APPROVED** [review]

**Review by Fdawgs — CHANGES_REQUESTED** [review]
How useful is this when there is GitHub Codespaces and github.dev available?

**Comment by pawlean** [comment]
> How useful is this when there is GitHub Codespaces and github.dev available?

Hey @Fdawgs 👋🏼 Pauline from Gitpod here. Great question! We have a full comparison which you can find [here](https://www.gitpod.io/vs/github-codespaces). 😄 Let me know if you have any further questions!

**Comment by Eomm** [comment]
@Fdawgs Internal discussion: https://github.com/orgs/fastify/discussions/35

**Review by Eomm — APPROVED** [review]

**Comment by pawlean** [comment]
@Eomm @Fdawgs It's great to see this merged! 

FYI: Fastify qualifies for our Open Source Plan, in partnership with the Linux Foundation [[1](https://www.gitpod.io/blog/linux-foundation-gitpod-partnership)] That is **2500 credits** (or 250 hours) of cloud development environments free of charge!

If you want to make use of this, you can apply directly [here](https://www.gitpod.io/discover/opensource). 🥳  Let me know if you have any questions!

**Pull request merged**